atlite/wind.py

```diff
@@ -90,3 +101,105 @@ def extrapolate_wind_speed(ds, to_height, from_height=None):
+    """
+    
+    try: 
+        # Get the turbine list
+        OEDB_URL = 'https://openenergy-platform.org/api/v0/schema/supply/tables/turbine_library/rows'
+        result = requests.get(OEDB_URL)
+    except requests.exceptions.RequestException as e:
+        logger.info(f"Connection to OEDB failed with:\n\n{str(e)}")
+        return None
+
+    # Convert JSON to dataframe for easier filtering
+    # Only consider turbines with power curves available
+    df = pd.DataFrame.from_dict(result.json())
+    df = df[df.has_power_curve]
+
+    # Check which information is provided for lookup and proceed accordingly
+    if turbine.get('id'):
+        logger.info(f"Searching for turbine in OEDB with id '{turbine['id']}'.")
+        ds = df[df.id == int(turbine['id'])]
+        
+    else:
+        if not all({isinstance(v, str)
+                    for k,v in turbine.items() if k in ['name', 'manufacturer']}
+                  ):
+            
+            logger.error("'name' and 'manufacturer' must be provided. "
+                         "Alternatively provide a turbine id from OEDB.")
+            return None
+        logger.info(f"Searching turbine power curve in OEDB database using "
+                    f"manufacturer '{turbine['manufacturer']}'"
+                    f"and name '{turbine['name']}'.")
+            
+        ds = df[df.name == turbine['name']]
+        ds = ds[ds.manufacturer == turbine['manufacturer']]
+
+    if len(ds) < 1 :
+        logger.info(f"No turbine found for {turbine}.")
+        return None
+    elif len(ds) > 2 :
+        logger.info(f"Provided information corresponds to more than one turbine. "
+                    f"Provide id to unambiguous lookup. {turbine}.")
+        return None
+    elif len(ds) == 1:
+        # Convert to series for simpliticty
+        ds = ds.iloc[0]
+
+    # convert power from kW to MW
+    power = np.array(json.loads(ds.power_curve_values)) / 1e3
+
+    turbineconf = {
+        "name": ds['name'].strip(),
+        "manufacturer": ds.manufacturer.strip(),
+        "source": f"Original: {ds.source}. Via OEDB {OEDB_URL}",
+        "HUB_HEIGHT": ds.hub_height.strip(),
+        "V": json.loads(ds.power_curve_wind_speeds),
+        "POW": power.tolist(),
+    }
+
+    if store_locally is True:
+        filename = "{m}_{n}.yaml".format(m=turbineconf['manufacturer'],
+                                        n=turbineconf['name'])
+        filename = filename.replace("/","_")
+        filename = filename.replace(" ","_")
+        filepath = utils.construct_filepath(os.path.join(config.windturbine_dir, filename))
+
+        with open(filepath, 'w') as turbine_file:
+            yaml.dump(turbineconf, turbine_file)
+            
+        logger.info(f"Turbine configuration downloaded to '{filepath}'.")
+
+    if ";" in turbineconf['HUB_HEIGHT']:
+        logger.info("Multiple HUB_HEIGHTS in dataset. Manual clean-up required.")
```

Comment:
```suggestion
        logger.warn(f"Multiple HUB_HEIGHTS in dataset ({turbineconf['HUB_HEIGHT']}). Manual clean-up required.")
```

Reply:
Pretty much all turbines in the [turbine_library](https://openenergy-platform.org/dataedit/view/supply/turbine_library) seem to have multiple hub_heights. Let's warn and take the median of all options for now. I'll try to find out, who is curating them and what the issue behind that is.

Reply:
Puh, that comes right out of the manufacturer's reference sheets: for instance, for Enercon: https://www.enercon.de/fileadmin/Redakteur/Medien-Portal/broschueren/pdf/ENERCON_Produkt_de_042017.pdf

Reply:
* Printing the hub heights makes sense
* `logging.warn(...)` is deprecated, use `logging.warning(...)` instead (see https://docs.python.org/3/library/logging.html#logging.Logger.warning)

Multiple hub heights are quite common and not that suprising. Depending on the project and location you would want / have to vary them.
I'll take the average but also preserve the original list... .

Reply:
+ I decided to go with the mean instead of median value.

Reply:
If you find the curating person, please tell per that I would appreciate a more consistent format :) 
e.g. ideally not `hub_height` entries with partially with a trailing semicolon: "123", "123;" "123; ", "123; 124;"

Reply:
Sure! :)